### ISSUE TITLE
Local symlinked repositories fail

### ISSUE THREAD

**Issue by BrianHenryIE** [BrianHenryIE/strauss#49]
Given a package that is in a directory on the local filesystem

```
"repositories": [
    {
        "type": "path",
        "url": "../bh-pdf-helpers"
    },
    ...
],
"require": {
    "brianhenryie/pdf-helpers": "dev-master",
    ...    
```

Error:

`The "/Users/brianhenry/Sites/my-project/vendor/Sites/bh-pdf-helpers/src/" directory does not exist.`

The problem is in [ComposerPackage constructor](https://github.com/BrianHenryIE/strauss/blob/e33c7670a4c2becb80c0031b26638da71157adb4/src/Composer/ComposerPackage.php#L105-L111).

This was introduced when supporting packages that install themselves to a folder different to their package name.

_Edit:_ Fix written but needs testing and thinking about.

**Comment by BrianHenryIE** [comment]
Does not occur with Phar. And integration test written to reproduce the issue did not fail.

**Comment by BrianHenryIE** [comment]
Fixed in [0.12.0](https://github.com/BrianHenryIE/strauss/releases/tag/0.12.0)
